public/app/types/serviceaccount.ts

```diff
@@ -0,0 +1,96 @@
+import { OrgRole, Unit } from '.';
+import { SelectableValue } from '@grafana/data';
+
+export interface OrgServiceAccount {
+  avatarUrl: string;
+  email: string;
+  lastSeenAt: string;
+  lastSeenAtAge: string;
+  login: string;
+  name: string;
+  orgId: number;
+  role: OrgRole;
+  serviceAccountId: number;
+}
+
+export interface ServiceAccount {
+  id: number;
+  label: string;
+  avatarUrl: string;
+  login: string;
+  email: string;
+  name: string;
+  orgId?: number;
+}
+
+export interface ServiceAccountDTO {
+  id: number;
+  login: string;
+  email: string;
+  name: string;
+  isGrafanaAdmin: boolean;
+  isDisabled: boolean;
+  isAdmin?: boolean;
+  isExternal?: boolean;
+  updatedAt?: string;
+  authLabels?: string[];
+  theme?: string;
+  avatarUrl?: string;
```

Comment:
question: will service accounts have avatars?

Reply:
I am pretty sure that we wanted them to have some kind of avatar eventually in the UI. So we should keep this